functions.php

```diff
@@ -374,3 +374,138 @@ function wp_get_list_item_separator() {
     return __( ', ' );
   }
 }
+
+
+// Update Registration form and functions
+add_action('login_head', function () {
+  ?>
+      <style>
```

Comment:
css should be in assets/css/site.scss file. you will need to install happy-scss-compiler version 1.3.9 , and set it up to compile from 'assets/css/' into 'assets/css-compiled/' folder, upon any chage.